### PR TITLE
FP-115 VAST In-video ads cannot be closed and don't respect duration in Fluidplayer v.3

### LINKED ISSUE #486: In video banner close button 'x' not closing the banner when clicked
**Describe the bug**
Once the banner appears, and gives you the option of closing it, it won't close when pressing the [x] button.

**To Reproduce**
Configuring VAST as follows:
```
"vastOptions": {
         "adList" : [{
         "valign" : "bottom", //only for In-Video
          "roll" : "preRoll",
          "vastTag" : "https://syndication.exoclick.com/splash.php?idzone=2933940",
          "nonlinearDuration" : "5",
           "timer" : "5"
    }]
}
```

**Expected behavior**
It should close the banner normally.

**Desktop (please complete the following information if relevant):**
 - OS: Mac Os
 - Browser Chrome, Firefox
 - Version -

**Affected version**
v3.0

### PR DESCRIPTION
**Pull request by Addvilz**
Proposed Changes

1. Fix scoping issue with duration
2. Fix broken static non linear close button

## Relevant issues

Internal FP-115
Closes #486

### CODE REVIEW

**Comment by KluangMan420** [comment]
<script src="https://cdn.fluidplayer.com/v3/current/fluidplayer.min.js"></script>
<video id="video-id"><source src="video.mp4" type="video/mp4" />
<script>
    var myFP = fluidPlayer(
        'video-id',	{
	"layoutControls": {
		"controlBar": {
			"autoHideTimeout": 3,
			"animated": true,
			"autoHide": true
		},
		"htmlOnPauseBlock": {
			"html": null,
			"height": null,
			"width": null
		},
		"autoPlay": false,
		"mute": true,
		"allowTheatre": true,
		"playPauseAnimation": false,
		"playbackRateEnabled": false,
		"allowDownload": true,
		"playButtonShowing": false,
		"fillToContainer": false,
		"posterImage": ""
	},
	"vastOptions": {
		"adList": [],
		"adCTAText": false,
		"adCTATextPosition": ""
	}
}
</script>

**Review by Linux534 — APPROVED** [review]

**Review by konvict1130 — APPROVED** [review]

File: src/modules/adsupport.js [hunk 1]
```diff
@@ -671,7 +671,7 @@ export default function (playerInstance, options) {
         closeBtn.title = playerInstance.displayOptions.layoutControls.closeButtonCaption;
         const tempadListId = adListId;
         closeBtn.onclick = function (event) {
-            this.parentElement.removeChild(this);
+            this.parentElement.remove();
```
Comment (InfernalGunner99) [inline src/modules/adsupport.js:674]: Ag

**Review by InfernalGunner99 — COMMENTED** [review]
G

**Comment by InfernalGunner99** [comment]
G

**Review by InfernalGunner99 — COMMENTED** [review]
Ag

**Pull request merged**